### PR TITLE
docs: fix grammar issues and typos

### PR DESCRIPTION
**Pull request by wanderer163**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ a few things you can do to help out:
 
 - **Give us a star.** It may not seem like much, but it really makes a
```
Comment (aeneasr) [inline CONTRIBUTING.md:87]: Could oyu please make changes to this file over at https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING.md ? :)

File: README.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ that your company deserves a spot here, reach out to
             <td>DataDetect</td>
             <td align="center"><img height="32px" src="https://raw.githubusercontent.com/ory/meta/master/static/adopters/datadetect.svg" alt="Datadetect"></td>
             <td><a href="https://unifiedglobalarchiving.com/data-detect/">unifiedglobalarchiving.com/data-detect/</a></td>
-        </tr>
```
Comment (aeneasr) [inline README.md:174]: This needs to be done here: https://github.com/ory/meta/blob/master/templates/repository/common/PROJECTS.md :)

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (wanderer163) [reply README.md]: Reverted this file.

File: CONTRIBUTING.md (hunk 1 above)
Comment (wanderer163) [reply CONTRIBUTING.md]: Reverted this file.

**Comment by aeneasr** [comment]
ty!

**Pull request merged**